### PR TITLE
Item v2/validator

### PR DESCRIPTION
**Pull request by ovekyc**
Add validator test code 
Edit validator routing for test

please check commit message

### CODE REVIEW

File: src/server/items/validator.js [hunk 1]
```diff
@@ -10,21 +10,20 @@ export default (req, res, next) => {
...
       statusCode: 400,
       message: 'wrong item Id'
     }));
   }
-  if (req.body.description && !(typeof req.body.description === 'string'
-    && validator.isLength(req.body.description, {
+  if (req.body.title && !(typeof req.body.title === 'string'
+    && validator.isLength(req.body.title, {
       min: MIN_DECSRIPSION_LENGTH,
       max: MAX_DECSRIPSION_LENGTH
```
Comment (hewonjeong) [inline src/server/items/validator.js]: Lets edit these const vars to 'MIN_TITLE_LENGTH' too.
And we have keyMaker module now, so we can refer const vars about making key to key Maker Module.
Lets move const vars to models.js file

File: src/server/items/validator.js [hunk 2]
```diff
@@ -53,7 +52,6 @@ export default (req, res, next) => {
       message: 'wrong item address'
     }));
   }
-
   if (req.body.createdDate && !(typeof req.body.createdDate === 'string'
     && validator.isDate(req.body.createdDate))) {
     return next(new APIError(new Error(), {
```
Comment (hewonjeong) [inline src/server/items/validator.js:57]: now we don't have to check createdDate and modifiedDate cause these are made in API func.

File: src/test/node/items/validator.js [hunk 3]
```diff
@@ -0,0 +1,424 @@
...
+  const expected = {
+    key: 'item-wv6mcsrb-5795ef07-d25c-42b2-8797-c242acaa5a9a'
+  };
+  const req = httpMocks.createRequest({
+    method: 'GET',
+    url: `/items/${expected.key}`,
+    params: {
+      id: `${expected.key}`
+    },
+    validatorTest: true
```
Comment (hewonjeong) [inline src/test/node/items/validator.js:27]: how about this?
`const req = httpMocks.createRequest({
    method: 'GET',
    url: '/items/:id',
    params: {
      id:`${key}`
    }
  });`
Cause our item-router is setted as ~~~/items/:id

File: src/server/items/validator.js (hunk 2 above)
Comment (ovekyc) [reply src/server/items/validator.js:57]: We still need this validation for 'put' function

Comment (hewonjeong) [reply src/server/items/validator.js:57]: I see. solved~!

**Comment by hewonjeong** [comment]
LGTM

**Comment by festinalente91** [comment]
LGTM :)

**Pull request merged**